Makefile

```diff
@@ -96,3 +96,8 @@ install: .env
 .PHONY: lint
 lint:
 	pre-commit run --all-files
+
+
+.PHONY: attach-django
+attach-django:
```

Comment:
What is this used for?

Reply:
If you add an `import ipdb; ipdb.set_trace()` somewhere you need to attach to the container in order for it to capture your keyboard input (`dc logs` just streams, but doesn't capture). So this makes it easier to attach to the django service.

Reply:
Ahh, got it!